src/Nest/Indices/Monitoring/IndicesStats/IndicesStats.cs

```diff
@@ -7,7 +8,12 @@ public class IndicesStats
 	{
 		[JsonProperty(PropertyName = "primaries")]
 		public IndexStats Primaries { get; set; }
+
 		[JsonProperty(PropertyName = "total")]
 		public IndexStats Total { get; set; }
+
+		[JsonProperty("shards")]
+		[JsonConverter(typeof(VerbatimDictionaryKeysJsonConverter<string, IndicesStats>))]
+		public IReadOnlyDictionary<string, ShardStats[]> Shards { get; set; } = EmptyReadOnly<string, ShardStats[]>.Dictionary;
```

Comment:
the types in the `JsonConverter` don't match the types in the property. Guessing one is incorrect?

Reply:
My bad. `JsonConverter` is incorrect.